### PR TITLE
Termination fixes

### PR DESCRIPTION
**Pull request by thijsc**
Make some changes in behaviour to make sure we implement this properly:

https://github.com/edenhill/librdkafka/wiki/Proper-termination-sequence

### CODE REVIEW

File: src/client.rs [hunk 1]
```diff
@@ -88,7 +88,6 @@ impl Drop for NativeClient {
         trace!("Destroy rd_kafka {:p}", self.ptr);
         unsafe {
             rdsys::rd_kafka_destroy(self.ptr);
-            rdsys::rd_kafka_wait_destroyed(1000);
```
Comment (fede1024) [inline src/client.rs]: Why was this removed?

Comment (thijsc) [inline src/client.rs]: See the commit message:

> It's not needed anymore in 0.9, see:
>
> https://github.com/edenhill/librdkafka/wiki/Proper-termination-sequence

Comment (fede1024) [inline src/client.rs]: I didn't notice that. Thanks!
